packages/zed-editor-preview-bin/zed-editor-preview-bin.pacscript

```diff
@@ -0,0 +1,39 @@
+pkgname="zed-editor-preview-bin"
+pkgdesc="A high-performance, multiplayer code editor from the creators of Atom and Tree-sitter (Preview release)"
+pkgver="0.155.1"
+repology=("project: zed-editor")
+gives="zed-editor-preview"
+license=("GPL-3.0-or-later" "AGPL-3.0-or-later" "Apache-2.0")
+arch=("x86_64" "aarch64")
+url="https://zed.dev"
+_channel="preview"
+source=("${url}/api/releases/${_channel}/${pkgver}/zed-linux-${AARCH}.tar.gz")
+sha256sums_x86_64=("afc3f27925af4c4d029d5bf5e420384fd4301c45aef7030d4ee8c4f4d61bbb6f")
+sha256sums_aarch64=("6962817f1b23b85e805e2da687b4fb718fe42a88740c4d834dca4c35e4089fa0")
+maintainer=("bibelin <balian1belin@yandex.ru>")
+
+# Desktop file should be renamed to dev.zed.Zed-Preview.desktop to show correct icon in GNOME on Wayland
+# Issue: https://github.com/zed-industries/zed/issues/16694
+_desktop="dev.zed.Zed-Preview.desktop"
+
+package() {
+  install -Dm755 "zed-${_channel}.app/bin/zed" -t "${pkgdir}/opt/${gives}/bin/"
+  install -Dm755 "zed-${_channel}.app/libexec/zed-editor" -t "${pkgdir}/opt/${gives}/libexec"
+  find "${srcdir}/zed-${_channel}.app/lib" -type "f" -exec install -Dm644 "{}" -t "${pkgdir}/opt/${gives}/lib/" \;
+  install -Dm644 "zed-${_channel}.app/share/icons/hicolor/1024x1024/apps/zed.png" "${pkgdir}/usr/share/icons/${gives}.png"
```

Comment:
```suggestion
  local size
  for size in 512x512 1024x1024; do
    install -Dm644 "zed-${_channel}.app/share/icons/hicolor/${size}/apps/zed.png" \
    "${pkgdir}/usr/share/icons/hicolor/${size}/apps/${gives}.png"
  done
```
I looked at the archive contents and saw there's two sizes

Reply:
Ah okay, I didn't thought it's important to include both